app/controllers/application_controller.rb

```diff
@@ -1,7 +1,7 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery
   layout :layout_by_resource
-  before_filter :set_effective_date
+  # before_filter :set_effective_date
```

Comment:
Minor nit... I'd just delete the line. I'd prefer to not check in commented code.